src/fsharp/fsc.fs

```diff
@@ -1887,18 +1918,29 @@ let main2(Args(tcConfig, tcImports, frameworkTcImports: TcImports, tcGlobals, er
                                    yield! bytes
                                | _ -> 
                                    failwith "unreachable: expected a local resource" |]
-            let sigDataFileName = (Filename.chopExtension outfile)+".fsdata"
-            File.WriteAllBytes(sigDataFileName,bytes)
+            //TODO instead of bytes array we can directly write to stream, less memory pressure
+            bytes |> emitAllBytes fsDataP
```

Comment:
FWIW it looks to me like useSignatureDataFile is always false and this is dead code, as is anything to do with fsData files.